docs/Typedefs.md

```diff
@@ -743,6 +743,19 @@ for authenticating and communicating with an individual API and the external mod
 | [weatherLocationId] | <code>number</code> |  |
 | [zip] | <code>string</code> | US Zip Code |
 
+<a name="FacilityFeedStatus"></a>
+
+## FacilityFeedStatus : <code>Object</code>
+**Kind**: global typedef  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| groupings | [<code>Array.&lt;FacilityGroupingStatus&gt;</code>](#FacilityGroupingStatus) | An array of field groupings associated with the feed |
+| id | <code>Number</code> |  |
+| key | <code>String</code> | The unique key for the feed |
+| status | <code>String</code> | The most recent status of the feed, e.g. "Healthy" |
+| updatedAt | <code>string</code> | ISO 8601 Extended Format date/time string |
+
```

Comment:
One thing that stands out to me is I think we are missing the `eventId`. We will need that to match it with the user subscription. (let me know if you have a different idea of how to handle it)

Reply:
Good point. I will add that to the API and then update this.